src/impl/tpls/KokkosBlas_tpl_spec.hpp

```diff
@@ -120,6 +120,18 @@ inline void cublas_internal_safe_call(cublasStatus_t cublasState,
 #define KOKKOS_CUBLAS_SAFE_CALL_IMPL(call) \
   KokkosBlas::Impl::cublas_internal_safe_call(call, #call, __FILE__, __LINE__)
 
+// This function converts KK transpose mode to cuBLAS transpose mode
+inline cublasOperation_t trans_mode_kk_to_cublas(const char kkMode[]) {
```

Comment:
Please use doxygen-style comments. For example, see:
https://github.com/kokkos/kokkos-kernels/blob/8381db0486674c1be943de23974821ddfb9e6c29/src/blas/KokkosBlas1_iamax.hpp#L53-L60

Reply:
Done. Thanks.